### DISCUSSION TITLE
Slash commands not appearing in Copilot chat in Visual Studio

### DISCUSSION THREAD

**Discussion by caitanya-3c** [community/community#78029] [Enterprise]
### Select Topic Area

Bug

### Body

Slash commands work (i.e. they appear when you type a "/")  in VSCode but they do not appear in the inline chat or normal chat in Visual Studio. According to this article it should be available in Visual Studio as well - https://learn.microsoft.com/en-us/visualstudio/ide/copilot-chat-context?view=vs-2022

Everything else Copilot related seems to be working fine.

Visual Studio 2022 Enterprise - Version 17.8.3
GitHub Copilot - Version 1.139.0.0
GitHub Copilot Chat - Version 0.1.1982.7773

**Comment by rheapatel23** [comment]
Hi, thank you for bringing this to our attention, our team is investigating this and will update you soon :) 

Thanks, 
Rhea - Product Manager | GitHub Copilot for Visual Studio

**Reply by caitanya-3c** [reply]
Any update on this?

**Comment by mike-clark-8192** [comment]
If you type part of a `/` command and then press backspace, the chat command "completion" popup starts showing up. 🤔

Example: type `/ee` in the [chat window](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-chat) then press backspace. You should see all the chat commands that have the letter 'e' in them.

**Comment by caitanya-3c** [comment]
This seems to be resolved for me now, for both inline and chat. I didn't do anything in particular my side, so I can only assume an update has now fixed it:
Visual Studio 2022 Enterprise - Version 17.8.5
GitHub Copilot - Version 1.155.0.0
GitHub Copilot Chat - Version 0.2.393.21236
